ChangeLog

```diff
@@ -9,6 +9,13 @@
 
 2015-04-13  Daniel Standage  <daniel.standage@gmail.com>
 
+   * scripts/sample-reads-randomly.py: use broken paired reader to provide paired-
```

Comment:
Also, make sure lines are wrapped before 80 characters, please; go ahead and fix Elmar Bucher's 4/13 entry, too, if you can.